containment-rpm-pxe.spec

```diff
@@ -28,6 +28,11 @@ BuildRequires:  filesystem
 Requires:       fdupes
 Requires:       libxml2-tools
 Requires:       perl-TimeDate
+%if 0%{?sle_version} >= 150300
```

Comment:
I am not sure if this will be true once MU is merged. Currently new python-kiwi is branched to 15.3 only because they need it there. But plan is, IIUC, that this new kiwi will be unbranched and go to SLE15 stream.

Reply:
See https://bugzilla.suse.com/show_bug.cgi?id=1184280

Reply:
Let me check with `150100`, as according to the MI for `python-kiwi`, only `>= SP1` will get the updated package.

Reply:
```
nothing provides kiwi-systemdeps-disk-images needed by containment-rpm-pxe, nothing provides kiwi-systemdeps-image-validation needed by containment-rpm-pxe
```
For both 4.1 and 4.0 (SP2 and SP1).

@aaannz am I missing anything? How is that the packages are not available? Will they become available only with the `python-kiwi` update, I guess?

Reply:
I am afraid so. We can merge this as is now, just take note that this will need fixing once new kiwi is available.

Reply:
Ok, then the plan is the following: 

- I changed to spec to require the packages for >= SP1.
- I will merge as soon as you approve
- I will push 0.2.6 only to Head for now
- I will push it to 4.1, 4.0 and Uyuni master only when the images start to fail, as that will mean that the python-kiwi was released

Reply:
Just for reference, I added the MI repos to https://build.suse.de/project/show/home:juliogonzalezgil:branches:Devel:Galaxy:Manager:4.1 and https://build.suse.de/project/show/home:juliogonzalezgil:branches:Devel:Galaxy:Manager:4.0 and now the images are building fine with the changes to the SPEC from this PR.